lib/msal-react-native-android-poc/blogPost.md

```diff
@@ -0,0 +1,125 @@
+
+At this point, I was about three weeks into my project; I had presented my BrownBag and started my project a week ahead of schedule. After I was able to implement the signIn method, the other methods, including signOut, getAccount, isSharedDevice, acquireTokenSilent, and acquireToken, were straightforward. I would implement a method, create a pull request, address reviews, merge, and repeat. My manager advised me to create multiple, short PRs instead of infrequent, larger PRs, which I felt sometimes slowed my progress down but also made it easier for my team to review. I noticed that in some of my PRs, my team would review some aspects that weren't caught in the previous PR, which supported my manager's thinking.
+
+I was about done with coding my project by the end of the fifth week, which was much earlier than expected. My six methods were working well, and my sample app was successfully showcasing them. My manager and I soon talked about what I wanted to do for the rest of my internship. I expressed that I was looking into adding functionality for multiple-account mode and was confident it could be finished in the remaining weeks. My manager said that I could definitely do that, but he also brought up another option. He said that wrapping up a project ends up taking longer than we usually think. While I have a completed project, it might be a good idea to spend a good amount of time polishing it so that it's ready for demos during the last week. After I finish that, he said I could potentially work on fixing a bug or adding a small feature to the actual MSAL JS library. As much as I was looking forward to adding another feature to my project, I couldn't pass up the opportunity to work with the team directly. And my project's main goal as a proof of concept was achieved, anyway: by creating a native module that bridges single-account mode methods, I proved that MSAL React Native is very much possible to implement.
+
+It took me another three weeks to write the necessary documentation, add little improvements, get my last PR reviewed (and address those reviews), and write this blog post. Tying up loose ends did take longer than expected, but I'm glad that I had the extra time to make sure my project looked great. 
+ 
+My completed proof of concept library allows React Native developers to utilize MSAL Android's single-account mode methods, scoped to AAD. The sample app demonstrates each of the six bridge methods in the library, using retrieved access tokens to make API calls to Microsoft Graph as an example. 
+
```

Comment:
Minor suggestion: This whole set of paragraphs are very long, may be subsection them based on the project flow can help readability.